testsuite/views.py

```diff
@@ -172,3 +154,29 @@ def initdb():
     ''' Initialize the database.
     '''
     db.create_all()
+
+
+def update_bloom_filter():
+    bloom_file_status = {}
+    bloom_file = './data/codes.bf'
+
+    try:
```

Comment:
See comment above; or more generally let's talk through ways to avoid the requirement that a developer be online to use/develop the test suite. (Could even be as simple as keeping the current code and adding an env variable to skip `update_bloom_filter` on startup.)